app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -62,5 +66,53 @@ data class OverdueAppointment(
           ORDER BY A.scheduledDate, A.updatedAt ASC
           """)
     fun appointmentsForFacility(facilityUuid: UUID, scheduledStatus: Appointment.Status, dateNow: LocalDate): Flowable<List<OverdueAppointment>>
+
+    /**
+     * FYI: IntelliJ's SQL parser highlights `isAtHighRisk` as an error, but it's not. This is probably
+     * because referencing column aliases in a WHERE clause is not SQL standard, but sqlite allows it.
+     */
+    @Query("""
+          SELECT P.fullName, P.gender, P.dateOfBirth, P.age_value, P.age_updatedAt, P.age_computedDateOfBirth,
+
+          BP.uuid bp_uuid, BP.systolic bp_systolic, BP.diastolic bp_diastolic, BP.syncStatus bp_syncStatus, BP.userUuid bp_userUuid,
+          BP.facilityUuid bp_facilityUuid, BP.patientUuid bp_patientUuid, BP.createdAt bp_createdAt, BP.updatedAt bp_updatedAt,
+
+          A.uuid appt_uuid, A.patientUuid appt_patientUuid, A.facilityUuid appt_facilityUuid, A.scheduledDate appt_scheduledDate, A.status appt_status,
+          A.cancelReason appt_cancelReason, A.remindOn appt_remindOn, A.agreedToVisit appt_agreedToVisit, A.syncStatus appt_syncStatus,
+          A.createdAt appt_createdAt, A.updatedAt appt_updatedAt,
+
+          PPN.uuid phone_uuid, PPN.patientUuid phone_patientUuid, PPN.number phone_number, PPN.phoneType phone_phoneType, PPN.active phone_active,
+          PPN.createdAt phone_createdAt, PPN.updatedAt phone_updatedAt,
+
+          (
+            CASE
+              WHEN MH.hasHadStroke = 1 THEN 1
+              WHEN BP.systolic > 160 AND BP.diastolic > 100
+                  AND (MH.hasDiabetes = 1 OR MH.hasHadKidneyDisease = 1)
+                  AND (P.dateOfBirth < :sixtyYearsDateOfBirth OR P.age_computedDateOfBirth < :sixtyYearsDateOfBirth)
+                  THEN 1
+              ELSE 0
+            END
+          ) AS isAtHighRisk
+
+          FROM Patient P
+
+          INNER JOIN Appointment A ON A.patientUuid = P.uuid
+          INNER JOIN BloodPressureMeasurement BP ON BP.patientUuid = P.uuid
+          LEFT JOIN PatientPhoneNumber PPN ON PPN.patientUuid = P.uuid
+          LEFT JOIN MedicalHistory MH ON MH.patientUuid = P.uuid
+
+          WHERE A.facilityUuid = :facilityUuid AND A.status = :scheduledStatus AND A.scheduledDate < :dateNow AND PPN.number IS NOT NULL
+          AND (A.remindOn < :dateNow OR A.remindOn IS NULL)
+
+          GROUP BY P.uuid HAVING max(BP.updatedAt)
+          ORDER BY isAtHighRisk DESC, A.scheduledDate, A.updatedAt ASC
+          """)
+    fun appointmentsForFacility(
+        facilityUuid: UUID,
+        scheduledStatus: Appointment.Status,
+        dateNow: LocalDate,
+        sixtyYearsDateOfBirth: LocalDate
```

Comment:
`sixtyYearsDateOfBirth` is something specific to a protocol for calculating whether someone is at high risk or not.

I feel that this age must be configurable and can be named to a more generic name like `bornBeforeDate` or something like that.

Reply:
Makes sense. I'll also add this to the config when I have more details about how we're categorizing high risk patients.

Reply:
Actually, doing `bornBeforeDate` -> `bornBefore`